install-run-ipfs.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/sh
+wget https://dist.ipfs.io/go-ipfs/v0.5.0/go-ipfs_v0.5.0_linux-amd64.tar.gz -O /tmp/go-ipfs_linux-amd64.tar.gz
```

Comment:
Consider upgrading straight to IPFS 0.6.0, as it has been [released](https://github.com/ipfs/go-ipfs/releases/tag/v0.6.0) some time ago. AFAIK it didn't introduce any breaking changes in the API.

Reply:
Done